src/tools/genpolicy/src/utils.rs

```diff
@@ -22,6 +22,13 @@ struct CommandLineOptions {
     )]
     config_map_file: Option<String>,
 
+    #[clap(
+        long,
+        num_args = 0..=std::usize::MAX,
+        help = "Optional Kubernetes YAML input file paths containing config resources such as ConfigMaps and Secrets"
```

Comment:
Users might have a hard time figuring out how to use this flag without further docs - see the big warning at https://docs.rs/clap/latest/clap/struct.Arg.html#method.num_args. Can we use `value_delimiter=','` and document it?

Reply:
I vote for:

#[clap(short, long, help = "Optional Kubernetes YAML input file path containing config resources such as ConfigMaps and Secrets)]
config_file: Option<Vec<String>>,

genpolicy doesn't have other arguments that support multiple values, and using

genpolicy -c file1.yaml -c file2.yaml 

seems reasonable enough to me.

Reply:
sgtm